x-pack/plugins/canvas/public/components/workpad_header/share_menu/utils.test.ts

```diff
@@ -16,10 +16,11 @@ const basePath = ({
   get: () => 'basepath/s/spacey',
   serverBasePath: `basepath`,
 } as unknown) as IBasePath;
+const kibanaVersion = '7.15,0';
```

Comment:
nit: appears to be a typo, might not be a big of a deal in a test, or may be intentional.

Reply:
061ef074413